docs/source/60_building_apps.md

```diff
@@ -256,6 +256,39 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s ShuffleRegionJoin API
+Another useful API implemented in ADAM is the ShuffleRegionJoin API, which joins two genomic datasets that contain overlapping regions. This primitive is useful for a number of applications including Variant calls and Feature mapping.
+
+To perform a ShuffleRegionJoin, add the following to your ADAM script:
+
+```dataset1.shuffleRegionJoin(dataset2)```
+
+Where dataset1 and dataset2 are GenomicRDDs. If you used the ADAMContext to read a genomic dataset into memory, this condition is met.
+
+ADAM has a variety of ShuffleRegionJoin types that you can perform on your data, and all are called in a similar way:
+
+``` scala
```

Comment:
I have a figure that we can use here (see below). I might suggest that instead of providing code snippets, we do the figure and a table, as a figure is worth a thousand words. ;) The table would show which joins are supported as broadcast joins and which are supported as shuffle joins. IIRC, everything is supported as a shuffle join.

![joins](https://cloud.githubusercontent.com/assets/3752466/22659423/700c8aae-ec52-11e6-8534-0e364f91700a.png)